README.md

```diff
@@ -102,6 +103,11 @@ If a satoshi is subsequently packaged up in an output of more than one satoshi,
 Due to the use of Tap Root in BTC, inscriptions are exposed in the input scripts. On BSV, they are written in outputs. 
 Due to this difference, Inscription IDs in 1SatOrdinals are stated in relation to the output of a transaction, and take the form of `<txid hex>_<vout>`.
 
+Only the first inscription in an output script is valid. Any subsequent inscriptions SHALL be ignored.
```

Comment:
I think this should be MUST ignored, or are ignored. Shall allows forking of stateful protocol depending on how indexers implement this recommendation.

Consider: spend 1000 brc mint to three outputs: 900, 100 as second inscription, 100. Ledgers have forked between indexers that respect SHALL and those that dont.

Reply:
Also would reword first inscription is valid as "Only first valid inscription envelope produces inscription on 1SatOrdinal."